aksel.nav.no/website/pages/eksempler/popover/no-flip.tsx

```diff
@@ -23,7 +23,7 @@ const Example = () => {
         flip={false}
         placement="right"
       >
-        <Popover.Content>
+        <Popover.Content style={{ maxWidth: "calc(50vw - 80px)" }}>
```

Comment:
Please help me test this in Safari. Should look something like this:
![image](https://github.com/user-attachments/assets/957aac22-d440-4080-a559-4e6c2c6801a8)

Reply:
Looks good :shipit: 
<img width="833" alt="Screenshot 2024-11-21 at 12 35 46" src="https://github.com/user-attachments/assets/c0750d34-6855-46ba-b170-fb47479f20a7">